
---
string str1
string str2
string str3